http_cache.rst

```diff
@@ -77,25 +77,15 @@ but is a great way to start.
 
     For details on setting up Varnish, see :doc:`/http_cache/varnish`.
 
-To enable the proxy, first create a caching kernel::
-
-    // src/CacheKernel.php
-    namespace App;
-
-    use Symfony\Bundle\FrameworkBundle\HttpCache\HttpCache;
-
-    class CacheKernel extends HttpCache
```

Comment:
Next example also uses the `CacheKernel` that was defined here, should that also be changed? I think the point here was to immediately show that you can create your own Kernel class and set default options in it with a later example.

Reply:
Not sure if this is a good change

Reply:
You right, I just figured out that my change imply to also change next example.
I guess that better to reject my change.